src/backend/functions/change-requests.ts

```diff
@@ -0,0 +1,37 @@
+import { Context } from 'aws-lambda';
+import { ChangeRequest } from 'utils/src/types/change-request-types';
```

Comment:
Your current import statement here is a file path reference, but in order to properly have the deployment automation bundle all the needed files for the AWS lambda build, it needs to be a package reference instead. 

Basically it needs to just be `'util'`, but if you don't understand the above statement, let me know and I'll send you some resources to learn about stuff.